.github/workflows/lint-workflows.yaml

```diff
@@ -157,3 +157,30 @@ jobs:
           for type in focus k8s-versions prs scheduled; do
             yamale -s ${type}-schema.yaml *-${type}.yaml;
           done
+
+  conformance-schema-validation:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@bd6b4b6205c4dbad673328db7b31b7fab9e241c0 # v4.6.1
+        with:
+          python-version: '3.10'
+      - run: pip install yamale
+      - name: Checkout code
+        uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab # v3.5.2
+        with:
+          persist-credentials: false
+          # hard-code the path instead of using ${{ github.repository }} to make sure it works for forked repo as well
+          path: src/github.com/cilium/cilium
+
+      - name: Validate schema of aws, azure and gke action files
+        shell: bash
+        run: |
+          for dir in aws azure gke;do
+            for name in k8s-versions test-config; do
+              file_base="src/github.com/cilium/cilium/.github/actions/${dir}/${name}"
+              if [ -f ${file_base}.yaml ];then
+                yamale -s ${file_base}-schema.yaml ${file_base}.yaml;
+              fi
+            done
+          done
+
```

Comment:
Ideally this would be handled in a separate PR on which all of #26356, #26351 and #26354 depend, but it's OK to just have the other PRs depend on this one as well. I think at the very least we should split this in a separate commit though.

Reply:
It was good that this part was highlighted because tests did not cover some of the files.
Fixed it, and separated it as a commit.